### ISSUE TITLE
More sophisticated preview of inactive numeral styles

### ISSUE THREAD

**Issue by davelab6** [graphicore/mdlFontSpecimen#15]
Some numeral features don't actually cause any changes from defaults.

Eg on https://graphicore.github.io/mdlFontSpecimen/ I see

<img width="336" alt="screen shot 2018-05-16 at 15 10 11" src="https://user-images.githubusercontent.com/261579/40146669-46ad2538-591b-11e8-9e10-98db98cd2d3f.png">

Perhaps these should be colored grey, or otherwise indicated that nothing is actually happening... Or, the sample should show some context where the overall textblock uses another numeral style, and then a child span applies this feature.

**Comment by graphicore** [comment]
Yeah, that's in general the problem that we know what features are there but we don't have meaningful sample texts. Fishing for substitutions made by a feature is possible, but cumbersome and would produce nonsensical samples in many cases. When we know the font, we could put in the sample texts somewhere into the specimen options or markup.

> Perhaps these should be colored grey, or otherwise indicated that nothing is actually happening

~~Hard to figure, I think there's no browser API that tells us if anything or what happened.~~ I misread you, yeah we could grey them out manually.

> Or, the sample should show some context where the overall textblock uses another numeral style, and then a child span applies this feature.

If this feature does nothing to numbers, why is it there in the first place?
![feature creep](http://assets.amuniversal.com/6117c9f06d6301301d80001dd8b71c47) (I thought this was funny in context of the nothing changing `lnum` feature. The request that is this issue is legit!)

I'm not sure if I get your proposal here. You mean the context would be `onum` and the sample would be not using `onum`, to imply what `lnum` would be doing if it would do anything?

We should maybe just remove the` lnum` tile?

**Comment by graphicore** [comment]
Oh, in this case, we know the font, so we could improve the sample, but it still does nothing. We should maybe just remove the `lnum` tile?

**Comment by graphicore** [comment]
Updates in my first reply
